lib/orbit-common/schema.js

```diff
@@ -494,6 +494,16 @@ var Schema = Class.extend({
     }
   },
 
+  linkPropertiesFor: function(type, link){
+    var model = this.models[type];
```

Comment:
I feel like this should be `var model = this.schemaFor(type);` but whatever, it's only 2 lines repeated ;)